src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random rnd = new Random();
+    private static final int MAX_VALUE = 20;
+
+    public Figure getRandomFigure() {
+        int numFigure = rnd.nextInt(5);
+        switch (numFigure) {
+            case 0:
+                return new Circle(ColorSupplier.getRandomColor(),
+                        getRandomDimension());
+            case 1:
+                return new Rectangle(ColorSupplier.getRandomColor(),
+                        getRandomDimension(), getRandomDimension());
+            case 2:
+                return new IsoscelesTrapezoid(ColorSupplier.getRandomColor(),
+                        getRandomDimension(), getRandomDimension(), getRandomDimension());
+            case 3:
+                return new RightTriangle(ColorSupplier.getRandomColor(),
+                        getRandomDimension(), getRandomDimension());
+            case 4:
+                return new Square(ColorSupplier.getRandomColor(),
+                        getRandomDimension());
+            default:
+                return null;
+        }
```

Comment:
We can create 5 private methods getRandomCircle(), getRandomSquare(), ... and call them in switch-case construction (private methods should follow after public as helper ones)